src/qt/rpcconsole.cpp

```diff
@@ -461,15 +461,30 @@ void RPCConsole::clear()
                     platformStyle->SingleColorImage(ICON_MAPPING[i].source).scaled(ICON_SIZE, Qt::IgnoreAspectRatio, Qt::SmoothTransformation));
     }
 
+    QFont fontMatch("Monospace");
```

Comment:
May not work according to https://stackoverflow.com/questions/1468022/how-to-specify-monospace-fonts-for-cross-platform-qt-applications#comment28709540_1835938